src/containerapp/azext_containerapp/java_component_decorator.py

```diff
@@ -137,6 +114,42 @@ def delete(self):
         except Exception as e:
             handle_raw_exception(e)
 
+    def set_up_component_type(self):
+        self.java_component_def["properties"]["componentType"] = self.get_argument_target_java_component_type()
+
+    def set_up_replicas(self):
+        if self.get_argument_min_replicas() is not None and self.get_argument_max_replicas() is not None:
+            self.java_component_def["properties"]["scale"] = {
+                "minReplicas": self.get_argument_min_replicas(),
+                "maxReplicas": self.get_argument_max_replicas()
+            }
+
+    def parse_configurations(self, configurations):
+        configuration_list = []
+        seen_properties = set()  # Track already seen property names
+
+        for pair in configurations:
+            key_val = pair.split('=', 1)
+            if len(key_val) != 2:
+                raise ValidationError("Java configuration must be in format \"<propertyName>=<value> <propertyName>=<value> ...\".")
+            property_name = key_val[0].lower()
```

Comment:
We should not change the customer input without any information.

Reply:
ok, removed the `lower()`